elements/bulbs-slick-slideshow/bulbs-slick-slideshow.test.js

```diff
@@ -0,0 +1,94 @@
+import { BulbsSlickSlideshow } from './bulbs-slick-slideshow';  // eslint-disable-line no-unused-vars
+import $ from 'jQuery';
+
+describe('<bulbs-slick-slideshow>', function () {
+  let parentElement;
+  let subject;
+
+  function attachSubject () {
+    parentElement = document.createElement('parent-element');
+    parentElement.appendChild(subject);
+
+    subject.attachedCallback();
+  }
+
+  beforeEach(() => {
+    subject = document.createElement('bulbs-slick-slideshow');
+  });
+
+  afterEach(() => {
+    parentElement.remove();
+  });
+
+  it('renders a <bulbs-slick-slideshow>', function () {
+    attachSubject();
+
+    expect(subject.tagName.toLowerCase()).to.eql('bulbs-slick-slideshow');
+  });
+
+  describe('#init', function () {
+    let navElement;
+
+    beforeEach(function () {
+      subject = document.createElement('bulbs-slick-slideshow');
+
+      navElement = document.createElement('div');
+      navElement.setAttribute('class', 'slider-nav');
+
+      subject.appendChild(navElement);
+
+      attachSubject();
+      
+      sinon.spy(subject.slideshow, 'slick');
+    });
+
+    it('inits the jQuery slick carousel with the correct stuff', function () {
+      let { initialSlide } = subject;
+
```

Comment:
stuck on this 👇 test 🤕 

I keep getting `AssertionError: expected slick to have been called with arguments`